k8s/service_controller/server/k8s/storage.go

```diff
@@ -120,7 +120,7 @@ func (kss *K8sServiceStorage) AddBroker(broker *model.ServiceBroker, catalog *mo
 	// versions:
 	// - name: v1alpha1
 	ksb := NewK8sSB()
-	ksb.Metadata = Meta{Name: broker.Name}
+	ksb.Metadata = Meta{Name: broker.GUID}
```

Comment:
I think this is right but just want to make sure, where is this info used? If its used internally and its required to be a constant even in the face of a SB rename then I agree.